alb_no_ssl_no_s3logs/sg.tf

```diff
@@ -22,10 +22,11 @@ resource "aws_security_group_rule" "sg_alb_http_ingress" {
 
 
 resource "aws_security_group_rule" "sg_alb_backend_egress" {
+  count                    = "${var.backend_sg_count}"
```

Comment:
can you apply the change I suggested please as we have it for example here?
https://github.com/skyscrapers/terraform-rds/blob/master/rds/security_group.tf#L15